ginga/rv/plugins/Compose.py

```diff
@@ -10,8 +10,9 @@
 
 **Usage**
 
-Start the ``Compose`` plugin from the "Operations" menu -- the tab should
-show up under "Dialogs".
+Start the ``Compose`` plugin from the "Operation->RGB" (below) or
+"Plugins->RGB" (above) menu. The tab should show up under the
+"Dialogs" tab in the viewer to the right as "IMAGE:Compose".
```

Comment:
To clarify a little further, the "IMAGE" in "IMAGE:Compose" is actually the channel name that the "Compose" local plugin is tied to. For example, if you start "Compose" from "DETECTOR1" channel name instead (there is a way to create new channels where you can provide useful names), it would appear as "DETECTOR1:Compose". Ways to potentially clarify this include but not limited to:

1. `"<channel name>:Compose"`
2. `"IMAGE:Compose", where "IMAGE" is the associated channel name`

I'll leave it to your good judgement. Thanks again for helping!